specification/dataprotection/resource-manager/Microsoft.DataProtection/DataProtection/BackupVaultResource.tsp

```diff
@@ -223,6 +258,58 @@ interface BackupVaultOperationResults {
   >;
 }
 
+/**
+ * Deleted backup vault operations (available from version 2025-09-01)
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-interface-requires-decorator" "DeletedBackupVaults interface uses manual routing for location-scoped resources"
+@added(Versions.v2025_09_01)
+interface DeletedBackupVaults {
+  /**
+   * Gets a deleted backup vault
+   */
+  @get
+  @route("/subscriptions/{subscriptionId}/providers/Microsoft.DataProtection/locations/{location}/deletedVaults/{deletedVaultName}")
```

Comment:
Why are you using this low-level representation of the API?  You should use templates for this